.gitmodules

```diff
@@ -13,3 +13,6 @@
 	url = https://github.com/naokiyokoyama/mask_rcnn_detectron2
 	branch = main
 	ignore = dirty
+[submodule "third_party/hand_object_detector"]
+	path = third_party/hand_object_detector
+	url = https://github.com/ddshan/hand_object_detector
```

Comment:
@zephirefaith  do we need to add hand-obj detector right now?
I suggest let's remove this as we are not using it?

Reply:
removed and moved to a separate PR - https://github.com/facebookresearch/spot-sim2real/pull/114